baselines/ppo1/run_soccer.py

```diff
@@ -13,27 +13,31 @@
 from core.soccer_env import SoccerEnv
 
 
-def train(env_id, num_timesteps, seed, save_model, load_model, model_dir):
-    from baselines.ppo1 import mlp_policy, pposgd_simple
+def train(env_id, num_timesteps, seed, save_model, load_model, model_dir, timesteps_per_actorbatch,
+        clip_param, ent_coeff, epochs, learning_rate, batch_size, gamma, lambd, exploration_rate, filename):
+    from baselines.ppo1 import kick_policy, pposgd_simple, reward_scaler
     rank = MPI.COMM_WORLD.Get_rank()
     U.make_session(num_cpu=1).__enter__()
     workerseed = seed + 10000 * rank
     set_global_seeds(workerseed)
     env = SoccerEnv(rank)
 
     def policy_fn(name, ob_space, ac_space):
-        return mlp_policy.MlpPolicy(name=name, ob_space=ob_space, ac_space=ac_space,
-                                    hid_size=64, num_hid_layers=2)
+        return kick_policy.KickPolicy(name=name, ob_space=ob_space, ac_space=ac_space,
```

Comment:
I changed this line to my case but maybe could be better maintain the MlpPolicy as default (It is easier for who is starting to use this code).

Reply:
Makes sense